modules/biddoBidAdapter.js

```diff
@@ -0,0 +1,92 @@
+import {registerBidder} from '../src/adapters/bidderFactory.js';
+import {BANNER} from '../src/mediaTypes.js';
+
+const BIDDER_CODE = 'biddo';
+const ENDPOINT_URL = 'https://ad.biddo.net/delivery/impress';
```

Comment:
Calling this endpoint from a test page returned `ERR_NAME_NOT_RESOLVED`. Is this set up yet?

Reply:
Hello! Thank you for review.
Our bid server production deployment is in progress.

Reply:
@llays This PR can't be approved and merged if the bid server isn't set up yet.

Reply:
@msm0504 Updated endpoint.

Reply:
@llays I'm still not getting a response from the endpoint.
```
https://ad.adopx.net/delivery/impress?ctype=div&pzoneid=379783&width=300&height=250
```